src/reynir_correct/errfinder.py

```diff
@@ -554,11 +554,23 @@ def VillaÍTölu(self, txt: str, variants: str, node: Node) -> AnnotationDict:
         subject = self._node_text(ch0)
         # verb_phrase = self._node_text(children[1])
         number = "eintölu" if "et" in variants else "fleirtölu"
+        # Find the verb
+        ch1node = cast(Node, ch1)
+        vptree: Optional[SimpleTree] = self._simple_tree(ch1node)
+        so: Optional[SimpleTree]  = None
```

Comment:
Extra space here - please run the Black formatter on the code

Reply:
Done